docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,73 @@
+networks:
```

Comment:
If we decide to add more compose files, maybe this file should be `ha-docker-compose.yaml`

Reply:
we can rename it then, at this point it is fine